File: app/src/main/java/com/rachelleignacio/listbucket/presentation/presenters/MainActivityPresenter.java
package com.rachelleignacio.listbucket.presentation.presenters;

import com.rachelleignacio.listbucket.domain.models.List;

/**
 * Created by rachelleignacio on 3/31/17.
 */

public interface MainActivityPresenter {

    interface View {
        void showListBucket();
        void showList(List list);
    }
}

File: app/src/main/kotlin/com/rachelleignacio/listbucket/presentation/presenters/MainActivityPresenter.kt
package com.rachelleignacio.listbucket.presentation.presenters

import com.rachelleignacio.listbucket.domain.models.List

/**
 * Created by rachelleignacio on 8/23/17.
 */
interface MainActivityPresenter {
    interface View {
        fun showListBucket()
        fun showList(list: List)
    }
}